pipelinewise/fastsync/commons/target_snowflake.py

```diff
@@ -148,7 +144,7 @@ def copy_to_table(self, s3_key, target_schema, table_name, size_bytes, is_tempor
         sql = f'COPY INTO {target_schema}."{target_table.upper()}" FROM \'@{stage}/{s3_key}\'' \
               f' FILE_FORMAT = (type=\'CSV\' escape=\'\\x1e\' escape_unenclosed_field=\'\\x1e\'' \
               f' field_optionally_enclosed_by=\'\"\' skip_header={int(skip_csv_header)}' \
-              f' compression=\'GZIP\' binary_format=\'HEX\')'
+              f' compression=\'GZIP\' binary_format=HEX)'
```

Comment:
just wondering, why the quotes are not required for `binary_format=HEX` but it's required for the other COPY options?

Reply:
Oh this, I removed the quotes from HEX because I run this same copy command manually on the snowflake console and it raised a warning that quotes are not neccessary for HEX. 
It has no impact, both ways work, looking at [copy documentation](https://docs.snowflake.com/en/sql-reference/sql/copy-into-table.html) file options are not quoted, so we can adjust the other options as well.

Reply:
I can rollback this change or do the same with other options, I'm fine with both :)

Reply:
that's fine thanks, whichever option you prefer. I was just curious. If both option work but quotes raise warnings maybe we can consider to remove quotes everywhere at some point

Reply:
Ok, I remove the quotes around `CSV` and `GZIP` in this commit https://github.com/transferwise/pipelinewise/pull/434/commits/02a7a97b32a4392e525644de876a4c211fada8ee